packages/textkit/src/layout/verticalAlign.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-restricted-syntax */
+
+/**
+ * Apply scaling and yOffset for verticalAlign 'sub' and 'super'.
+ *
+ * @param  {Object} layout options
+ * @param  {Object} attributed string
+ * @return {Object} attributed string
+ */
+const verticalAlignment = () => attributedString => {
+  attributedString.runs.forEach(run => {
+    const { attributes } = run;
+    const { verticalAlign, fontSize } = attributes;
+
+    if (verticalAlign === 'sub') {
+      attributes.fontSize = fontSize * 0.75;
+      attributes.yOffset = -0.2;
+      attributes.characterSpacing = (fontSize - attributes.fontSize) / -2.45;
```

Comment:
what is `-2.45`? should this parameter be gotten from a font?

Reply:
If I just applied the new fontSize here the character spacing would still use the spacing of the previous fontSize so it has to be adjusted. I came to a factor of -2.45 by trying out different parameters and comparing results.

But this will not be needed if I change the implementation to just setting vertical alignment as a numeric value.

Reply:
Could you please remove other font modifications and keep only hardcoded `sub`/`super`?

I agree that it is not so useful, but it will follow the spec 
https://w3c.github.io/csswg-drafts/css2/#valdef-vertical-align-sub

Reply:
> Could you please remove other font modifications and keep only hardcoded `sub`/`super`?
> 
> I agree that it is not so useful, but it will follow the spec https://w3c.github.io/csswg-drafts/css2/#valdef-vertical-align-sub

Yes will do, I just read your other comment about making it numeric 😊

I'll build the fontSize logic in my side of the application, setting it that way also has the advantage of not having to correct the character spacing.